packages/web/test/src/lib/myalgowallet-mode.ts

```diff
@@ -1,27 +1,29 @@
-import algosdk, { Account, Transaction } from "algosdk";
+import algosdk, { Account, LogicSigAccount, Transaction } from "algosdk";
 import assert from "assert";
 
 import { MyAlgoWalletSession, testnetURL, types } from "../../../src";
 import { algoexplorerAlgod, getSuggestedParams } from "../../../src/lib/api";
 import { HttpNetworkConfig } from "../../../src/types";
 import { MyAlgoConnectMock } from "../../mocks/myalgowallet-mock";
-import { receiverAccount, senderAccount } from "../../mocks/tx";
+import { createLsigAccount, receiverAccount, senderAccount } from "../../mocks/tx";
 
 describe("Webmode - MyAlgo Wallet test cases ", () => {
 	let connector: MyAlgoWalletSession;
 	let sender: Account;
 	let receiver: Account;
+	let lsigAccount: LogicSigAccount
 	const walletURL: HttpNetworkConfig = {
 		token: "",
 		server: testnetURL,
 		port: "",
 	};
 	const algodClient: algosdk.Algodv2 = algoexplorerAlgod(walletURL);
 
-	beforeEach(() => {
+	before(async () => {
 		sender = senderAccount;
 		receiver = receiverAccount;
```

Comment:
this way the tests cases will depend on each other

Reply:
oh okay will update